Gems/Atom/RPI/Code/Include/Atom/RPI.Edit/Shader/ShaderOptionValuesSourceData.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/std/string/string.h>
+#include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/Memory/SystemAllocator.h>
+
+namespace AZ
+{
+    class ReflectContext;
+
+    namespace RPI
+    {
+        using ShaderOptionValuesSourceData = AZStd::unordered_map<Name/*optionName*/, Name/*valueName*/>;
```

Comment:
I was following the pattern of other classes in this library that use "SourceData" as a suffix to indicate json serialized content. Though this is a bit different because there is no source file for just shader option values.